src/qt/diagnosticsdialog.cpp

```diff
@@ -161,18 +161,13 @@ int DiagnosticsDialog::VerifyAddNode() {
 }
 
 void DiagnosticsDialog::VerifyClock() {
-    QHostInfo info = QHostInfo::fromName("pool.ntp.org");
+    QHostInfo NTPHost = QHostInfo::fromName("pool.ntp.org");
     udpSocket = new QUdpSocket(this);
 
-    connect(udpSocket, SIGNAL(readyRead()), this, SLOT(clkFinished()));
+    connect(udpSocket, SIGNAL(stateChanged(QAbstractSocket::SocketState)), this, SLOT(clkStateChanged(QAbstractSocket::SocketState)));
```

Comment:
I assume the `clk` prefix is supposed to be an abbreviation of "clicked". Most likely copied from other signal handlers. Change these to `on` as they are not click triggered. Example, `onStateChanged`.